packages/service-clients/azure-client/src/AzureClient.ts

```diff
@@ -59,6 +67,32 @@ const azureClientFeatureGates = {
 	"Fluid.Container.ForceWriteConnection": true,
 };
 
+/**
+ * Feature gates required to support runtime compatibility when V1 and V2 clients are collaborating
+ */
+const azureClientV1CompatFeatureGates = {
```

Comment:
@andre4i are there other configs that need to go in here for op virtualization (e.g. grouped batches)?

Reply:
"Fluid.ContainerRuntime.CompressionDisabled": true,
"Fluid.ContainerRuntime.CompressionChunkingDisabled": true

we're not enabling grouped batching by default just yet, but preemptively:

"Fluid.ContainerRuntime.DisableGroupedBatching": true

https://github.com/microsoft/FluidFramework/tree/main/packages/runtime/container-runtime/src/opLifecycle#disabling-in-case-of-emergency

But are we doing this right now? Last time I was in a discussion regarding these I thought we would be communicating them to our customers and having them manage the feature gates depending on their deployment/saturation numbers. Have we changed approaches to only one feature gate?

Reply:
I haven't been in those discussions.  So I was throwing this out there.  I'm going to revert the bit about the uber `"Fluid.Compatibility.V2Only"` gate and just put the GC ones in here.  Handing back to @scottn12 for broader strategy.

He assigned a task to me to make sure GC would be compatible between v1/v2, hence me doing this PR at all.